backend/src/migrations/1543523539612_init.js

```diff
@@ -0,0 +1,98 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+    pgm.createTable('users', {
+        email: { type: 'varchar(254)', primaryKey: true},
+        first_name: {type: 'varchar(30)'},
+        last_name: {type: 'varchar(150)'},
```

Comment:
Limit 256? I think it's better to use powers of 2

Reply:
Passwords will be of uniform length, because we store only hashes and they are always of the same length. 128 bits is enough for SHA512 hash, but yes we can reserve 256 symbols, for future use.